### ISSUE TITLE
issue lists (homepage, /issues) should pull in milestone name

### ISSUE THREAD

**Issue by miketaylr** [webcompat/webcompat.com#1784]
It shows labels currently, when we remove the "status" labels, we'll lose that info. So we need to read from the milestone.

And update fixtures.

**Comment by miketaylr** [comment]
Dunno if we need to design anything for this, or just make sure it always appears in the right-most position.

**Comment by karlcow** [comment]
<img width="1394" alt="capture d ecran 2017-09-08 a 10 20 28" src="https://user-images.githubusercontent.com/505230/30193415-94042734-947f-11e7-8d42-e3b9cb9250b5.png">

**Comment by karlcow** [comment]
Let's open a new issue. It currently fails because of #1796

**Comment by karlcow** [comment]
#1796 has been fixed. Let's go back to this

**Comment by karlcow** [comment]
The markup for this is now 

```html
<span class="wc-IssueList-status js-Issue-status">
          <a href="/issues?q=milestone:sitewait" class="wc-Labels" data-remotename="sitewait" title="Status : sitewait">sitewait</a>
        </span>
```

The state class color should be handled in #1757

Currently `wc-IssueList-status js-Issue-status` does not exist in the CSS and this should be styled too. Probably. It doesn't have to be the same style than labels as we escaped this. 

@magsout ideas.

**Comment by karlcow** [comment]
It looks like this for now.

<img width="475" alt="capture d ecran 2017-09-08 a 12 03 47" src="https://user-images.githubusercontent.com/505230/30195709-d7829e60-948d-11e7-97ce-0e48222551f9.png">

I will open a new issue

**Comment by miketaylr** [comment]
Fixed in #1803.
